src/easycurl.cpp

```diff
@@ -467,24 +467,6 @@ void wxEasyCurl::SetProxyAddress(const wxString &proxy)
 	gs_curlProxyAddress = proxy;
 }
 
-static std::string wstr2str(const wchar_t *s)

```

Comment:
It seems as if this function is used on line 533 of the same file, and that removing this function causes an unsolved reference.  Is this function defined elsewhere?

Reply:
Sorry, you're right. It's only used on Windows. I've just pushed a new change that (as minimally as possible) eliminates wstr2str() by initialising a std::string with iterators. If you agree, the rest of the changes on this branch should now be able to be merged.